src/CSModule.sol

```diff
@@ -85,6 +87,9 @@ contract CSModule is
     uint64 private _depositableValidatorsCount;
     uint64 private _nodeOperatorsCount;
 
+    // TODO: Place instead of publicRelease.
```

Comment:
We plan to keep public release since it also limits the number of keys per-operator